spec/features/staff/users_can_view_a_submission_spec.rb

```diff
@@ -31,7 +31,7 @@
       expect(page.response_headers["Content-Type"]).to include("text/csv")
 
       header = page.response_headers["Content-Disposition"]
-      expect(header).to match(/#{submission.description}/)
+      expect(header).to match(/#{ERB::Util.url_encode(submission.description)}/)
```

Comment:
Does the `submission.description` used in this example contain a string that would trigger the truncation issue?

Reply:
It has a space in it so yes I think so

Reply:
Maybe I should make the test explicitly check for `Legacy%20Submission` which is the actual string 🤔

Reply:
Yeah, checking for a concrete value rather than some dynamically produced value usually makes me more confident in what the test is doing.